src/components/Searchbar/Autocomplete.js

```diff
@@ -326,13 +387,22 @@ export default function Autocomplete({
             return autocompleteState.query !== '' ? (
               <div key={`source-${index}`} className="aa-Source">
                 {collection.source.sourceId === 'content' && (
-                  <span>Content</span>
+                  <Box padding="xs" fontWeight="600" color="#27272E99">
+                    Content
+                  </Box>
+                )}
+                {collection.source.sourceId === 'pages' && (
+                  <Box padding="xs" fontWeight="600" color="#27272E99">
```

Comment:
Can we use a theme var or pass down currentColor for hover states on the icon color?